framework/src/index.ts

```diff
@@ -138,6 +138,6 @@ export { AggregateCommit } from './engine/consensus/types';
 export { BFTHeights } from './engine/bft/types';
 export { BFTParameters } from './engine/bft/schemas';
 export { aggregateCommitSchema } from './engine/consensus/certificate_generation/schema';
-export { computeCertificateFromBlockHeader } from './engine/consensus/certificate_generation/utils';
+export { computeUnsignedCertificateFromBlockHeader as computeCertificateFromBlockHeader } from './engine/consensus/certificate_generation/utils';
```

Comment:
this we could actually rename rather than alias?

Reply:
somehow when I renamed symbol on vsCode it got renamed as alias